alert-engine/src/test/scala/io/phdata/pulse/alertengine/AlertEngineCliParserTest.scala

```diff
@@ -79,7 +79,7 @@ class AlertEngineCliParserTest extends FunSuite {
     assertResult(49152)(cliParser.smtpPort())
     assertResult("silenced-applications.txt")(cliParser.silencedApplicationsFile())
     assertResult(
-      "master1.valhalla.phdata.io/solr,master2.valhalla.phdata.io/solr,master3.valhalla.phdata.io/solr")(
+      "${HOSTNAME1}/solr,${HOSTNAME2}/solr,${HOSTNAME3}/solr")(
```

Comment:
No need to use (what looks) string interpolation here, just use 'hostname' or the like